models/ed/R/model2netcdf.ED2.R

```diff
@@ -643,11 +647,25 @@ put_T_values <- function(yr, nc_var, out, lat, lon, begins, ends, ...){
   
   
   # ----- define ncdf dimensions
-  
+  # setup output time and time bounds
+  sub_date_range <- seq(begins, by = "day", length.out = length(lubridate::yday(begins):ends))
+  sub_jdates <- lubridate::yday(lubridate::floor_date(seq(sub_date_range[1], sub_date_range[length(sub_date_range)],length.out= length(out[[1]]))))
+  iter_per_day <- length(sub_jdates[sub_jdates==lubridate::yday(sub_date_range[2])])  # look at day 2 to determine iterations per day (i.e. model timestep)
+  #iter_per_day <- length(out[[1]])/ends    ### does not work for part of a year!!
+  rm(sub_jdates)
+  step <- seq(0, 0.99, 1/iter_per_day)
+  sub_dates <- rep(sub_date_range,each=iter_per_day)
+  jdates <- lubridate::yday(sub_dates)
+  day_repeats <- as.vector(base::table(jdates)) # number of model outputs per day, e.g. 48 for half-hourly
+  tvals <- (jdates+step)-1
```

Comment:
If I'm interpreting this correctly (and that statement suggests a need for more comments) you're creating a time sequence by creating a day sequence and a subdaily sequence and then adding them together. This seems overly complicated. Is there a reason that neither `seq(begin_date,end_date,by=timestep)` nor `seq(begin_date,end_date,length)` worked?

Reply:
So in this I am attempting to use the info we have when running the put_out() functions istem wrote to generate a full range of run dates (from the specific year being processed), figure our the output timestep, then basically convert date to fractional DOY from 0-365/366 to use for the nc time as well as to create the time_bounds....

I know I went there a whole bunch of iterations to try and make this work, will need to look again to remind myself why I didnt do it that way.  Also, I think day_repeats is not used now anyway and needs to be deleted.  Let me check

Reply:
@mdietze I clean up that part of the code in the latest push and hopefully simplified